### DISCUSSION TITLE
Display directly after updating using the document method" Internal Server Error＂

### DISCUSSION THREAD

**Discussion by cdorsat** [mediacms-io/mediacms#920] [Q&A]
For example, if the title is updated using the document method, it will be displayed directly after updating."
Internal Server Error". What is the reason? How to solve this situation?

#########################################################
Environment: Debian 11.01
Installation mode: stand-alone installation

**Comment by mgogoulos** [comment]
`if the title is updated using the document method` not sure what you mean here. can you provide more detailed steps? 
also, do a `tail -f logs/*` before you run the action and see what is the error message
